codeworld-server/src/Util.hs

```diff
@@ -296,3 +299,53 @@ removeDirectoryIfExists dirName =
     handleExists e
         | isDoesNotExistError e = return ()
         | otherwise = throwIO e
+
+loadDumpedTree :: BuildMode -> UserId -> IO DirTree
+loadDumpedTree bm uid = do
```

Comment:
The convention so far is to use `mode` for variables containing a `BuildMode`.

Reply:
Ok.